models/image.model.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const Connection = require('./connection')
+const Room = require('../models/room.model')
+
+const connection = new Connection()
+const images = connection.sequelize.define(
```

Comment:
@Bymatt10 Por favor utiliza mayúsculas
```suggestion
const Images = connection.sequelize.define(
```